cmd/frr-monitoring/main.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
Do we want to call this `frr-exporter` to be more in line with common naming in the prometheus ecosystem? Or is that not accurate?

Reply:
renamed it to frr-exporter.